tests/test_transpilers_general.py

```diff
@@ -45,46 +45,52 @@ def custom_circuit():
 def special_connectivity(connectivity):
     """Return a TII harware chip connectivity as a networkx graph"""
     if connectivity == "21_qubits":
-        Q = [f"q{i}" for i in range(21)]
+        Q = []
```

Comment:
Labels might be nice for drawing it, if they have a meaning, but: are you drawing this?

If not, we might save a bunch of lines (and `Q`s) just using `int`s :)